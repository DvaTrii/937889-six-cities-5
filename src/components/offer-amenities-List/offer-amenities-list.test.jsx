import React from "react";
import renderer from "react-test-renderer";
import OfferAmenitiesList from "./offer-amenities-list";


it(`Should OfferAmenitiesList render properly`, () => {
  const amenities = [
    `Wifi`, `Parking`, `Dishwasher`
  ];

  const tree = renderer
    .create(
        <OfferAmenitiesList
          amenities={amenities}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
