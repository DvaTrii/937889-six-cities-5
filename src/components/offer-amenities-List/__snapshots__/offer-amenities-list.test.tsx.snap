// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferAmenitiesList render properly 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    Wifi
  </li>
  <li
    className="property__inside-item"
  >
    Parking
  </li>
  <li
    className="property__inside-item"
  >
    Dishwasher
  </li>
</ul>
`;
